Source/WebKit/WebProcess/Network/WebResourceLoader.cpp

```diff
@@ -131,7 +131,7 @@ void WebResourceLoader::willSendRequest(ResourceRequest&& proposedRequest, IPC::
         }
     }
 
-    coreLoader->willSendRequest(WTFMove(proposedRequest), redirectResponse, [this, protectedThis = WTFMove(protectedThis), completionHandler = WTFMove(completionHandler)] (ResourceRequest&& request) mutable {
+    coreLoader->willSendRequest(WTFMove(proposedRequest), redirectResponse, [this, protectedThis = Ref { *this }, completionHandler = WTFMove(completionHandler)] (ResourceRequest&& request) mutable {
```

Comment:
Why does this fix a safer C++ warning? Is it a flaw in the analyzer? Seems like it's removing an optimization and doesn't fix a lifetime bug. Of course it's not an important optimization, but I don't fully understand.

Reply:
Yes, this is to silence the static analyzer. 3 out of 5 look like false positives and I have reported them to @rniwa. Trying to get the bot back to green since someone recently unskipped this file.